sim/midas/src/main/scala/midas/stage/phases/CreateParametersInstancePhase.scala

```diff
@@ -13,13 +13,12 @@ import java.io.{File}
 
 case class ConfigParametersAnnotation(p: Parameters) extends NoTargetAnnotation with Unserializable
 
-class CreateParametersInstancePhase extends Phase with PreservesAll[Phase] {
-
-  override val prerequisites = Seq(Dependency(midas.stage.Checks))
-
-  // This copies the rocketChip get config code, but adds support for looking up a config class
-  // from one of many packages
-  def getConfigWithFallback(packages: Seq[String], configNames: Seq[String]): Config = {
+object CreateParametersInstancePhase {
```

Comment:
Might as well call this object `CreateParametersFromConfigString`, and make `getParameters` an `apply` method, since this is not a companion objection of `CreateParametersInstancePhase`.

Reply:
Great idea, thanks!